### PR TITLE
Microverse-Project-Review

### PR DESCRIPTION
**Pull request by vaderonline**
Added Previous Proejct

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!-- 
+
+Hello Camper!
+
+Please read the README below in the JS Editor before beginning. Feel free to delete this message once you have read it. Good luck and Happy Coding! 
+
+- The freeCodeCamp Team
```
Comment (vaderonline) [inline home.html:7]: Please Remove unuseful comments from the code to make your code more readable and clean

File: home.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <figcaption id="img-caption"> Mohandas Karamchand Gandhi (/ˈɡɑːndi, ˈɡændi/;<br>GAHN-dee; 2 October 1869 – 30 January 1948), also known as Gandhi, was an Indian lawyer, anti-colonial nationalist  and political ethicist<br> who employed nonviolent resistance to lead the successful campaign for India's independence from British rule,<br>and to later inspire movements for civil rights and freedom across the world.
+    </figcaption>
+  </figure>
+  <section id="tribute-info">
+    <h3 id="headline"> Information About Mahatma Gandhi </h3> 
+    <p> Mohandas Karamchand known as Mahatma or "Great-Souled" Gandhi was born in Porbandar, the capital of a small <br> principality in what is today the state of Gujarat in Western India, where his father was prime minister. His mother was <br>a profoundly religious Hindu. She and the rest of the Gandhi family belonged to a branch of Hinduism in which <br>non-violence and tolerance between religious groups were considered very important. His family background has later<br> been seen as a very important explanation of why Mohandas Gandhi was able to achieve the position he held in Indian society.</p>
```
Comment (vaderonline) [inline home.html:33]: If you want to add <tag> in your comment please wrap it with 2 backticks (the button right below the Escape button on the keyboard) and kindly always add the reason why the reviewee had to add the changes.

**Review by vaderonline — COMMENTED** [review]
# Changes Require ♻️ 

Please Check  my comments below and make all the required changes

File: home.html (hunk 1 above)
Comment (omar-muhamad) [reply home.html:7]: - [ ] Please add the reason why the reviewee had to do your required changes in this case you can say:
`Please Remove unuseful comments from the code to make your code more readable and clean`

File: home.html (hunk 2 above)
Comment (omar-muhamad) [reply home.html:33]: - [ ] If you want to add `<tag>` in your comment please wrap it with 2 backticks (the button right below the Escape button on the keyboard) and kindly always add the reason why the reviewee had to add the changes.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @vaderonline,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You use inline comments in your review.

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @vaderonline  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Good work implementing the changes from the previous reviewer. ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- It would be awesome if you backed up your commits with emojis 😎 . They make reviews more enjoyable 😄  and they bring out certain expressions words cannot. 👍

**Pull request merged**